apps/verticalhome/style/css/app.css

```diff
@@ -104,3 +99,38 @@ body.edit-group {
   font-weight: 300;
 }
 
+#blinknotif{
+  position: fixed;
+  left: 0px;
+  bottom: 3.2rem;
+  right: 0px;
+  background-color: #555;
+  color: #FFF;
+  z-index: 99;
+  content: "";
+  clear: both;
+  display: block;
+  text-align: left;
+  height: 3.2rem;
+  font-size: 1.3rem;
+  color: #fff;
+  display: none;
+  padding: 0.3rem 1rem;
+}
+
+#main-screen-header{
+  display: none;
+  background-color: rgb(175, 175, 175);
+  text-align: center;
+  color: rgb(255, 255, 255);
+  font-size: 1.5rem;
+  padding: 0.8rem 0px;
+  font-style: italic;
+  position: fixed;
+  top: 0px;
```

Comment:
I personally find 0 cleaner than 0px. Why add extra bytes when you don't need to.